test/GithubApi.Authentication.test.js

```diff
@@ -12,11 +12,7 @@ describe('Github Api Test', () => {
     it('Via OAuth2 Tokens by Header', async () => {
       const response = await axios.get(
         `${urlBase}/repos/${githubUserName}/${repository}`,
-        {
-          headers: {
-            Authorization: `token ${process.env.ACCESS_TOKEN}`
-          }
-        }
+        { access_token: process.env.ACCESS_TOKEN }
```

Comment:
this test should be using the Authorization header

Reply:
I'm sorry, I do not understand the question

Reply:
I will be reviewing the other files included in this PR during weekend

Reply:
ok thank you. I already changed it.